documentation/certificates.md

```diff
@@ -0,0 +1,91 @@
+Streisand PKI
+=============
+
+Streisand includes a role responsible for PKI certificate generation,
+separated into two distinct tasks:
+
+ - CA/Server
+ - Client(s)
+
+The first of which will generate a certificate authority, and a server 
+certificate signed by the newly minted certificate authority.
+
+The latter task is responsible for generating client side certificate
+(using the same CA) used for authenticting connecting VPN clients.
+
+Each VPN service that leverages PKI generates its own CA/Server and 
+client certificates.
+
+Infrastructure
+--------------
+
+The overall PKI infrasture assumes the following layout:
+
+```
+Web Root CA: 
+   \__ gateway HTTPS cert (unless LE; see below)
+
+OpenConnect CA: 
+   \__ OpenConnect server cert
+   \__ OpenConnect client 1
+   \__ OpenConnect client ...
+   \__ OpenConnect client n
+
+OpenVPN CA: 
+   \__ OpenVPN server cert
+   \__ OpenVPN client 1
+   \__ OpenVPN client ...
+   \__ OpenVPN client n
+
+IPsec-IKEv2 CA (upcoming):
```

Comment:
I think it's probably less confusing if you leave this out for now since it's upcoming